src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -552,9 +552,9 @@ void EnGo_InitSnow(EnGoEffect effect[ENGO_SNOW_EFFECT_COUNT], Vec3f pos) {
         effect->pos.y += 56.0f;
 
         // Generate a +-15 degree rotational velocity
-        effect->rotVelocity.x = (Rand_ZeroOne() - 0.5f) * (f32)0x1554;
-        effect->rotVelocity.y = (Rand_ZeroOne() - 0.5f) * (f32)0x1554;
-        effect->rotVelocity.z = (Rand_ZeroOne() - 0.5f) * (f32)0x1554;
+        effect->angularVelocity.x = (Rand_ZeroOne() - 0.5f) * (f32)0x1554;
```

Comment:
```suggestion
        effect->angularVelocity.x = (Rand_ZeroOne() - 0.5f) * 0x1554;
```
Not sure if better, though still out curiosity are the explicit casts needed?

Reply:
No, they're not needed. I removed them